### PR TITLE
Fix ie loading bug

### LINKED ISSUE #365: Js error for ScriptCache
We just start using this library, but got "loaderCB01563745391429 is not a function" error for IE11 on our sentry log.

### PR DESCRIPTION
**Pull request by dannyjlaurence-exp**
Closes #365 

This PR has two things:
* Fix script bug
* Use yarn instead of npm
* Replace "." with "bash" to better support cygwin
* Checking in the result of building? I was kind of confused when building produced different results in the dist/folder

The specific fix for the loaderCB<numbers> bug was pretty straightforward: The `handleResult` callback in the ScriptCache.js was being used both `onLoad` and as the callback to the Google script being loaded. On Chrome, the google callback runs first reliably and in IE/Edge onLoad runs first reliably - so in IE the window.loaderCB function that the Google Script was trying to call was already cleaned up.

To fix, I just naively sent a different named state to the `handleResult` function, and also returned on resolve and reject. I'm a little unclear on if this will cause any other issues - but for what we use this project for - it works fine.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,23 +1,23 @@
 .PHONY: publish dev test example
 
 dev:
-	npm run dev
+	yarn dev
```
Comment (auser) [inline Makefile]: Let's keep this with `npm` as it's more universal still at this stage.

File: package.json [hunk 2]
```diff
@@ -18,14 +18,14 @@
...
-    "preversion": ". ./scripts/prepublish.sh",
+    "prepublish": "bash ./scripts/prepublish.sh",
+    "preversion": "bash ./scripts/prepublish.sh",
     "dev": "NODE_ENV=development ./node_modules/hjs-webpack/bin/hjs-dev-server.js",
     "build": "NODE_ENV=production webpack",
     "publish_pages": "gh-pages -d public/",
     "lint": "eslint ./src",
     "lintfix": "eslint ./src --fix",
-    "testonly": "NODE_ENV=test mocha $npm_package_options_mocha",
+    "testonly": "NODE_ENV=test mocha --require scripts/mocha_runner -t rewireify src/**/__tests__/**/*.js",
```
Comment (auser) [inline package.json]: Why are we using `rewireify` here?

**Comment by dannyjlaurence-exp** [comment]
Thanks! I missed this email. I'll follow your feedback soon.

**Comment by auser** [comment]
Merged and published

**Comment by BenSwennen** [comment]
Hi, it looks like this https://github.com/fullstackreact/google-maps-react/pull/395/files#diff-f7444d992b0a47df772a034af14b92648da3121d4a9af3fd58fd2b074546916dR76 never got published? Or am I missing something? I'm still getting the loaderCBXXXX error on IE and Edge on version 2.0.6.

**Pull request closed without merging**